README.md

```diff
@@ -22,22 +21,22 @@ install a mamba environment
 ```
 conda create -c conda-forge --override-channels mamba -n mamba
 ```
- 
+
 activate this mamba environment
 ```
 conda activate mamba
 ```
- 
+
 install the improver environment using mamba
 ```
-mamba create -c conda-forge python=3.7 improver -n improver
+mamba create -c conda-forge python=3 improver -n improver
```

Comment:
I've run this against the current conda package for improver. An environment is build using python 3.9.23, which seems to allow the improver code to work with the various pinned versions of numpy, iris, etc. that the package currently mandates. Once we update this package I assume python 3.12 will be used and the expected new packages.